### PR TITLE
Rename idxs/idx to index/i/indices

### PR DESCRIPTION
**Pull request by luciaquirke**
- I know this is a bit psycho
- index/indices is slightly more conventional and I read it faster

### CODE REVIEW

**Review by valedan — APPROVED** [review]
I don't have a strong preference either way here, but seems reasonable! 😄

File: tests/unit/maze_transformer/training/test_dataset.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import torch
```
Comment (valedan) [inline tests/unit/maze_transformer/training/test_dataset.py:1]: Did you mean to include this file or was it part of your other PR?

Comment (luciaquirke) [reply tests/unit/maze_transformer/training/test_dataset.py:1]: Nope, cheers!

**Comment by rusheb** [comment]
> I know this is a bit psycho

Small readability PRs like this are awesome imo - keep them coming! 🦀

**Pull request merged**